inference/core/workflows/execution_engine/core.py

```diff
@@ -25,64 +36,87 @@ def init(
         prevent_local_images_loading: bool = False,
         workflow_id: Optional[str] = None,
     ) -> "ExecutionEngine":
-        if init_parameters is None:
-            init_parameters = {}
-        compiled_workflow = compile_workflow(
+        requested_engine_version = _retrieve_requested_execution_engine_version(
             workflow_definition=workflow_definition,
-            init_parameters=init_parameters,
         )
-        return cls(
-            compiled_workflow=compiled_workflow,
+        engine_type = _select_execution_engine(
```

Comment:
should this variable be called `envine_version`?

Reply:
it returns class of engine to be used matching version - I believe this is ok as is

Reply:
name suggests there are many types of engine, where in fact we have many versions but not types